modules/tinymce/src/plugins/link/test/ts/browser/OpenLinkContextMenuTest.ts

```diff
@@ -0,0 +1,60 @@
+import { FocusTools, Keyboard, Keys, TestStore } from '@ephox/agar';
+import { afterEach, describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import LinkPlugin from 'tinymce/plugins/link/Plugin';
+
+describe('browser.tinymce.plugins.link.OpenLinkContextMenuTest', () => {
+  const store = TestStore<string>();
+
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'link',
+    toolbar: 'link',
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce',
+    image_caption: true
+  }, [ LinkPlugin ], true);
+
+  // Capture the href links being clicked
+  document.addEventListener('click', (e) => {
```

Comment:
This isn't properly removed you could use sugar here it's DomEvent and use the unbinder of that. THis should also be done in a hook so before/after not just like this for everything regardless if the test is executed or not.

- [ ] Make event registration proper on before/after hooks and with a cleanup